includes/Traits/Types.php

```diff
@@ -92,7 +92,8 @@ public function get_allowed_mime_types() : array {
 				'image/jpg',
 				'image/gif',
 			],
-			'audio' => [], // todo: support audio uploads.
+			// TODO: Update once audio elements are supported.
+			'audio' => [],
```

Comment:
Why are we not adding the mime types here?

Reply:
IIUC, that would allow dragging & dropping audio files into the editor for upload. Not something we'd want to support right now.

Reply:
It wouldn't allow for it be uploaded. 

https://github.com/google/web-stories-wp/blob/3e4903ec5b34c31ec0d3d691234ee14b0a7f3b95/assets/src/edit-story/app/uploader/useUploader.js#L47-L57

Reply:
😢 Okay, worth a try.